rewriter/rewriter.js

```diff
@@ -120,7 +107,16 @@ for (var i = 2; i < process.argv.length; i++) {
   }
 }
 
-loadRuleSets();
+var hstsFilename = path.join(__dirname, 'hsts.xml');
+try {
+  fs.statSync(hstsFilename);
+} catch (e) {
+  console.error('hsts.xml not found. Run hsts-transform.js.');
```

Comment:
Any reason not to just output to `hsts.xml` instead of `stdout` from `hsts-transform.js`?